src/main/java/com/radixdlt/crypto/ECDSASignature.java

```diff
@@ -1,20 +1,20 @@
-/*
```

Comment:
Any idea why git is doing this?  The contents of the comment seem to be the same?

Reply:
Oh, I know.  Line endings.  Bah humbug.